device-types/Seagate Systems/Exos-CORVAULT-4U106.yaml

```diff
@@ -0,0 +1,55 @@
+---
+manufacturer: Seagate Systems
+model: Exos CORVAULT 4U106
+slug: seagate-systems-exos-corvault-4u106
+part_number: R4106I2500T002
+u_height: 4
+is_full_depth: true
+description: '[Seagate Exos CORVAULT 4U106](https://www.seagate.com/products/storage/data-storage-systems/corvault/4U106/)'
+power-ports:
+  - name: PSU 1
+    type: iec-60320-c20
+  - name: PSU 2
+    type: iec-60320-c20
+interfaces:
```

Comment:
The official way to model storage controllers is by modeling the controller as its own device, then creating `device-bays` with the chassis device.